sharding/data_modeling.rst

```diff
@@ -24,12 +24,77 @@ If your situation resembles either of these cases then the next step is to decid
 Distributing by Tenant ID
 =========================
 
-The multi-tenant architecture uses a form of hierarchical database modeling to distribute queries across nodes in the distributed cluster. The top of the data hierarchy is known as the *tenant id*, and needs to be stored in a column on each table. Citus inspects queres to see which tenant id they involve and routes the query to a single physical node for processing, specifically the node which holds the data shard associated with the tenant id. Running a query with all relevant data placed on the same node is called :ref:`colocation`.
+The multi-tenant architecture uses a form of hierarchical database modeling which allows Citus to execute queries entirely within single worker nodes. This allows greater efficiency, full SQL coverage, and foreign key support. The top of the data hierarchy is known as the *tenant id*, and needs to be stored in a column on each table. Citus inspects each query to see which tenant id it involves and routes the query to a single physical node for processing, specifically the node which holds the data shard associated with the tenant id.
 
-The first step is identifying what constitutes a tenant in your app. Common instances include company, account, organization, or customer. The column name will be something like :code:`company_id` or :code:`customer_id`. Examine each of your queries and ask yourself: would it work if it had additional WHERE clauses to restrict all tables involved to rows with the same tenant id? Queries in the multi-tenant model are usually scoped to a tenant, for instance queries on sales or inventory would be scoped within a certain store.
+Here is an example of two tables and their shards. Each box is a shard whose color represents which physical node contains it. Green shards are stored together on one physical node, and blue on another.  Notice how a join query between Accounts and Campaigns would have all the necessary data together on one node when restricting both tables to the same account_id. Matching the relevant shards of two tables in this way across all nodes is called *table co-location*.
```

Comment:
Could we make the first sentence more specific and transitionary?

The following diagram shows an example multi-tenant data model with two tables, where each table is sharded on account_id. Each box in this diagram is a shard ...